src/Symfony/Component/HttpFoundation/Tests/FileBagTest.php

```diff
@@ -60,6 +60,19 @@ public function testShouldSetEmptyUploadedFilesToNull()
         $this->assertNull($bag->get('file'));
     }
 
+    public function testShouldRemoveEmptyUploadedFilesForMultiUpload()
+    {
+        $bag = new FileBag(array('file' => array(
+            'name' => array(''),
```

Comment:
Is this really the structure that is used when multiple files are submitted?

Reply:
@xabbuh Yes.

Reply:
You can try yourself with the example script I posted above. Just add `var_export($_FILES)` somewhere.

Reply:
For the records, that's also explicitly described here: http://php.net/manual/en/features.file-upload.multiple.php

Reply:
I can't reproduce the issue, this `FileBag` with those values returns 

![file-bag](https://user-images.githubusercontent.com/2028198/30429912-437bc234-9927-11e7-81de-ccc54cd91bce.png)

according to:

https://github.com/symfony/symfony/blob/12f1239565f94a4fc9d4593190f7bc59c2dd0743/src/Symfony/Component/HttpFoundation/FileBag.php#L86-L87

Reply:
See test case also:

https://github.com/symfony/symfony/blob/77653d15ee75db672824d92025a1752a2ab7ea6a/src/Symfony/Component/Form/Tests/Extension/Core/Type/FileTypeTest.php#L99-L110